services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceController.java

```diff
@@ -239,6 +244,30 @@ public void start() {
     LOGGER.info("Controller is started.");
   }
 
+  private void initializeSystemSchema(Admin admin) {
+    String systemStoreCluster = multiClusterConfigs.getSystemSchemaClusterName();
+    if (!multiClusterConfigs.isParent() && multiClusterConfigs.isZkSharedMetaSystemSchemaStoreAutoCreationEnabled()
+        && multiClusterConfigs.getControllerConfig(systemStoreCluster)
+            .isSystemSchemaInitializationAtStartTimeEnabled()) {
+      UpdateStoreQueryParams metadataSystemStoreUpdate =
+          new UpdateStoreQueryParams().setHybridRewindSeconds(TimeUnit.DAYS.toSeconds(1)) // 1 day rewind
+              .setHybridOffsetLagThreshold(1)
+              .setHybridTimeLagThreshold(-1) // Explicitly disable hybrid time lag measurement on system store
+              .setWriteComputationEnabled(true)
+              .setPartitionCount(1);
```

Comment:
This seems to be a default setting for updating metadataSystemStore that appears in multiple places, e.g. VeniceHelixAdmin. Maybe it's good to put it as a util function instead of defining it every time?

Reply:
Will move it to a single place. Thanks.